src/main/java/com/terraformersmc/modmenu/gui/widget/DescriptionListWidget.java

```diff
@@ -333,20 +327,25 @@ public void renderScrollBar(BufferBuilder bufferBuilder, Tessellator tessellator
 			}
 
 			RenderSystem.setShader(GameRenderer::getPositionColorProgram);
-			bufferBuilder.begin(VertexFormat.DrawMode.QUADS, VertexFormats.POSITION_COLOR);
-			bufferBuilder.vertex(scrollbarStartX, this.getBottom(), 0.0D).color(0, 0, 0, 255).next();
-			bufferBuilder.vertex(scrollbarEndX, this.getBottom(), 0.0D).color(0, 0, 0, 255).next();
-			bufferBuilder.vertex(scrollbarEndX, this.getY(), 0.0D).color(0, 0, 0, 255).next();
-			bufferBuilder.vertex(scrollbarStartX, this.getY(), 0.0D).color(0, 0, 0, 255).next();
-			bufferBuilder.vertex(scrollbarStartX, q + p, 0.0D).color(128, 128, 128, 255).next();
-			bufferBuilder.vertex(scrollbarEndX, q + p, 0.0D).color(128, 128, 128, 255).next();
-			bufferBuilder.vertex(scrollbarEndX, q, 0.0D).color(128, 128, 128, 255).next();
-			bufferBuilder.vertex(scrollbarStartX, q, 0.0D).color(128, 128, 128, 255).next();
-			bufferBuilder.vertex(scrollbarStartX, q + p - 1, 0.0D).color(192, 192, 192, 255).next();
-			bufferBuilder.vertex(scrollbarEndX - 1, q + p - 1, 0.0D).color(192, 192, 192, 255).next();
-			bufferBuilder.vertex(scrollbarEndX - 1, q, 0.0D).color(192, 192, 192, 255).next();
-			bufferBuilder.vertex(scrollbarStartX, q, 0.0D).color(192, 192, 192, 255).next();
-			tessellator.draw();
+			bufferBuilder = tessellator.begin(VertexFormat.DrawMode.QUADS, VertexFormats.POSITION_COLOR);
+			bufferBuilder.vertex(scrollbarStartX, this.getBottom(), 0.0F).color(0, 0, 0, 255);
+			bufferBuilder.vertex(scrollbarEndX, this.getBottom(), 0.0F).color(0, 0, 0, 255);
+			bufferBuilder.vertex(scrollbarEndX, this.getY(), 0.0F).color(0, 0, 0, 255);
+			bufferBuilder.vertex(scrollbarStartX, this.getY(), 0.0F).color(0, 0, 0, 255);
+			bufferBuilder.vertex(scrollbarStartX, q + p, 0.0F).color(128, 128, 128, 255);
+			bufferBuilder.vertex(scrollbarEndX, q + p, 0.0F).color(128, 128, 128, 255);
+			bufferBuilder.vertex(scrollbarEndX, q, 0.0F).color(128, 128, 128, 255);
+			bufferBuilder.vertex(scrollbarStartX, q, 0.0F).color(128, 128, 128, 255);
+			bufferBuilder.vertex(scrollbarStartX, q + p - 1, 0.0F).color(192, 192, 192, 255);
+			bufferBuilder.vertex(scrollbarEndX - 1, q + p - 1, 0.0F).color(192, 192, 192, 255);
+			bufferBuilder.vertex(scrollbarEndX - 1, q, 0.0F).color(192, 192, 192, 255);
+			bufferBuilder.vertex(scrollbarStartX, q, 0.0F).color(192, 192, 192, 255);
+			try {
+				builtBuffer = bufferBuilder.end();
+				BufferRenderer.drawWithGlobalProgram(builtBuffer);
+				builtBuffer.close();
+			}
+			catch (Exception ignored) { }
```

Comment:
Nitpick: braces should be on the same lines as keywords.

```suggestion
			try {
				builtBuffer = bufferBuilder.end();
				BufferRenderer.drawWithGlobalProgram(builtBuffer);
				builtBuffer.close();
			} catch (Exception e) {
				// Ignored
			}
```